specs/Standards/SourceMetadata.md

```diff
@@ -2,11 +2,7 @@
 
 ## [NEP-330](https://github.com/near/NEPs/blob/master/neps/nep-0330.md)
 
-:::caution
-This is part of proposed spec [NEP-330](https://github.com/near/NEPs/blob/master/neps/nep-0330.md) and subject to change.
-:::
```

Comment:
I feel this scary disclaimer is not necessary anymore. This NEP is well-reviewed by Contract Standards Working Group and well-received by the community.